spec/lib/miq_automation_engine/service_methods/miq_ae_service_tenant_quota_spec.rb

```diff
@@ -4,8 +4,8 @@ module MiqAeServiceTenantQuotaSpec
   describe MiqAeMethodService::MiqAeServiceTenantQuota do
     let(:settings) { {} }
     let(:tenant) { Tenant.create(:name => 'fred', :domain => 'a.b', :parent => default_tenant) }
-    let(:cpu_quota) { TenantQuota.create(:name => "max_cpu", :unit => "int", :value => 2, :tenant_id => tenant.id) }
-    let(:storage_quota) { TenantQuota.create(:name => "stor", :unit => "GB", :value => 160, :tenant_id => tenant.id) }
+    let(:cpu_quota) { TenantQuota.create(:name => "cpu_allocated", :unit => "int", :value => 2, :tenant_id => tenant.id) }
+    let(:storage_quota) { TenantQuota.create(:name => "storage_allocated", :unit => "GB", :value => 160, :tenant_id => tenant.id) }
 
```

Comment:
@mkanoor Please review this change. I added a validation on "name" being one of ["cpu_allocated", "mem_allocated", "storage_allocated", "vms_allocated", "templates_allocated"] to the TenantQuota class. That broke this test. So I updated the quota names you had to match the ones in the validation.